crates/sound/src/stream.rs

```diff
@@ -89,7 +89,7 @@ type Result<T> = std::result::Result<T, Error>;
 ///         |              |           |          |         |
 ///         |              |<----------|          |         |
 /// ```
-#[derive(Debug, Default, Copy, Clone)]
+#[derive(Debug, Default, Copy, Clone, PartialEq, Eq)]
```

Comment:
why we need this change?

Reply:
I added based on a comment here https://github.com/virtio-sound/vhost-device/pull/32#discussion_r1325627215. However, I think that comment is now outdated. I will drop this change.

Reply:
BTW I agree with that comment, my question was more about why we need both. If not needed for this fix, I agree on remove the entire change.